apps/dashboard/src/hooks/use-sync-workflow.tsx

```diff
@@ -4,7 +4,13 @@ import { ConfirmationModal } from '@/components/confirmation-modal';
 import { showToast } from '@/components/primitives/sonner-helpers';
 import { PromoteSuccessToast } from '@/components/promote-workflow/promote-success-toast';
 import { useEnvironment } from '@/context/environment/hooks';
-import { WorkflowListResponseDto, WorkflowOriginEnum, WorkflowResponseDto, WorkflowStatusEnum } from '@novu/shared';
+import {
```

Comment:
```suggestion
import type {
```

Reply:
Will do that in bulk in separate PR + will try to add the rule. Now its throwing some errors when I do that.